group_project_v2/stage.py

```diff
@@ -631,7 +632,7 @@ def load_peer_feedback(self, request, suffix=''):  # pylint: disable=unused-argu
         return webob.response.Response(body=json.dumps(results))
 
     def do_submit_review(self, submissions):
-        peer_id = submissions["review_subject_id"]
+        peer_id = int(submissions["review_subject_id"])
```

Comment:
@Kelketek yes, it is coming from client-side. Usually it comes as string. Since it comes from client it might be anything (paranoid mode :)). If your question is actually "what if it is not parseable as string?", this is a json handler wrapped in catch-all exception providing proper error message to client.